src/apps/tools/OswAppFitnessStats.cpp

```diff
@@ -0,0 +1,67 @@
+#include "./apps/tools/OswAppFitnessStats.h"
+#include "./apps/watchfaces/OswAppWatchfaceDual.h"
+#include "./apps/watchfaces/OswAppWatchfaceFitness.h"
+
+#include <gfx_util.h>
+#include <osw_app.h>
+#include <osw_hal.h>
+#include <osw_pins.h>
+
+void OswAppFitnessStats::showFitnessTracking() {
+  OswHal* hal = OswHal::getInstance();
+
+  hal->gfx()->setTextMiddleAligned();
+  hal->gfx()->setTextCenterAligned();
+
+  hal->gfx()->setTextSize(2);
+  hal->gfx()->setTextCursor(DISP_H / 2, 60);
+  hal->gfx()->setTextColor(rgb565(255, 255, 255));
+  hal->gfx()->print("Fitness stats");
+
+  const char* type[] = {"Step", "Kcal", "Distance", "Total"};
+  int fitnessGoal[] = {OswConfigAllKeys::stepsPerDay.get(), OswConfigAllKeys::distPerDay.get(),OswConfigAllKeys::kcalPerDay.get(),0};
+  fitnessGoal[3]=(fitnessGoal[0]+fitnessGoal[1]+fitnessGoal[2])/3;
+  uint32_t fitnessValue[] = {hal->environment->getStepsToday(),OswAppWatchfaceFitness::calculateDistance(hal->environment->getStepsToday()),OswAppWatchfaceFitness::calculateKcalorie(hal->environment->getStepsToday()),0};
+  fitnessValue[3] = (fitnessValue[0] + fitnessValue[1]+fitnessValue[2])/3;
+
+  uint16_t fitnesColor[] = {ui->getInfoColor(),ui->getWarningColor(),ui->getDangerColor(),ui->getPrimaryColor()};
+  uint8_t x_point = 50;
```

Comment:
I need to clean up the variable names.

Reply:
Sooo, this is still in progress?

Reply:
We just need to check the device. I checked the compilation and logic with my brain several times without any problems. However, the dirty variable name and coarse code that i worked on remain. We need to sort this out.